cmd/list/list.go

```diff
@@ -23,6 +31,16 @@ func New(vp *viper.Viper) *cobra.Command {
 
 	listCmd.AddCommand(
 		newNodeCommand(vp),
+		newNamespacesCommand(vp),
 	)
 	return listCmd
 }
+
+func jsonOutput(buf io.Writer, v interface{}) error {
+	bs, err := json.MarshalIndent(v, "", "  ")
+	if err != nil {
+		return err
+	}
+	_, err = fmt.Fprintln(buf, string(bs))
```

Comment:
This was how it was already written, I just moved which file this function was in. I believe the goal is to get a newline at the end so that when it's logged it's not on the same line as the prompt.